clang/test/SemaSYCL/work-group-size-hint-host.cpp

```diff
@@ -1,19 +1,28 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
 
```

Comment:
Also here.   Please add a comment describing the test.

The name of the test: work-group-size-hint-host.cpp could be changed to something else.   I suggest check-work-group-size-hint.cpp (the word 'host' led me to first check if were testing host behavior here.)

Also, since this attribute is not restricted at the definition level, could you add a test that checks -fsycl-is-host behavior with this attribute?  I presume we will want it ignored or warned.

Reply:
Fixed it, thanks!